hr_timesheet_employee_allowed/views/account_analytic_line_view.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_timesheet_line_custom_inherit" model="ir.ui.view">
+        <field name="name">view.analytic.timesheet.line.inherit</field>
+        <field name="model">project.task</field>
+        <field name="inherit_id" ref="hr_timesheet.view_task_form2_inherited"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='timesheet_ids']/tree/field[@name='user_id']" position="after">
+                <field invisible="1" name="timesheet_line_user_id"/>
+            </xpath>
```

Comment:
Please add also to the main Tree view, accessed through Timesheets / Timesheet menu.